### PR TITLE
(#2190153) pstore: fix crash and forward dummy arguments instead of NULL

### PR DESCRIPTION
**Pull request by msekletar**
Resolves: #2190153

### CODE REVIEW

**Comment by dtardon** [comment]
@jamacku Apparently bot has eaten the bug reference in the description.

File: .github/advanced-commit-linter.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+policy:
+  cherry-pick:
+    upstream:
+      - github: systemd/systemd
+      - github: systemd/systemd-stable
+    exception:
+      note:
+        - rhel-only
```
Comment (jamacku) [inline .github/advanced-commit-linter.yml:8]:
```suggestion
        - rhel-only
        - RHEL-only
```

**Comment by jamacku** [comment]
> @jamacku Apparently bot has eaten the bug reference in the description.

Sorry, `^` and `$` are a part of regex and should not be added to the comment. I'll update GA.

**Review by dtardon — APPROVED** [review]
LGTM

**Pull request merged**